portfolio/src/main/webapp/style.css

```diff
@@ -167,8 +164,61 @@ h1{
     font-size:1.8vw;
 }
 
-.project-container a :hover{
+.project-container a:hover{
     opacity:0.6;
 }
 
+.socials{
+    position:relative;
+    text-align: center;
+
+}
+.social{
```

Comment:
social and socials are very close in name, and this can lead to confusion. Try renaming one of these. E.g. socialLink, or socialList, etc